cypress/integration/admin.js

```diff
@@ -59,5 +75,28 @@ describe('Sourcebooks admin', function() {
         
         check.click('[type="Submit"]');
         check.click('[href="/projects"]');
+    })*/
+
+    it('Admin should log time', function() {
+        var task = Math.random().toString(36).substring(2, 15)
+
+        var hours = (Math.round(Math.random() * 23) + Math.random()).toFixed(2)
+        
+
+
+        check.click('[href="/time-logging"]');
```

Comment:
according to page object model all locators should be moved out of test case files, but here I see all of them